src/main/java/uk/gov/hmcts/darts/common/task/AutomatedTask.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.hmcts.darts.common.task;
```

Comment:
Wondering if the `task` package should be up one level and the various subpackages for service, exception also be moved there?